src/cpu/core_full.cpp

```diff
@@ -63,6 +63,7 @@ typedef PhysPt EAPoint;
 
 Bits CPU_Core_Full_Run(void) {
 	FullData inst;	
+	inst.rm_eaa = 0;
 	while (CPU_Cycles-->0) {
```

Comment:
Looks like the safest place to do this; right when the core starts.
Looking at the FullData struct, in theory we could set default values for them?

``` c++
struct FullData {
	Bitu entry;
	Bitu rm;
	EAPoint rm_eaa;
	Bitu rm_off;
	Bitu rm_eai;
	Bitu rm_index;
	Bitu rm_mod;
	OpCode code;
	EAPoint cseip;
#ifdef WORDS_BIGENDIAN
	union {
		Bit32u dword[1];
		Bit32s dwords[1];
		Bit16u word[2];
		Bit16s words[2];
		Bit8u byte[4];
		Bit8s bytes[4];
		} blah1,blah2,blah_imm;
...
```

Gotta love the `blah`, `blah` variables :-)

Reply:
Opted not to touch the struct itself to avoid any chance of turning it into non-POD struct.